controller/create_or_update_jobs.py

```diff
@@ -91,19 +92,19 @@ def create_jobs(job_request):
         validate_job_request(job_request)
         project_file = get_project_file(job_request)
 
-        with tracer.start_as_current_span("create_jobs.load_pipeline"):
+        with time_for_span("duration_ms.load_pipeline", span):
```

Comment:
Good idea to include the units in the name, that's considered best practice AIUI.

However, the convention for naming for otel attrs would I think would be `load_pipeline.duration_ms`, i.e. operation first, details later. We'd then put any other load_pipline related things underneath it (e.g. `load_pipeline.action_count=n`) to group all load_pipeline things together. It's just an organisational convention, though.

There's lots of examples here: https://opentelemetry.io/docs/specs/semconv/general/attributes/

Also, I don't think that time_for_span is measuring ms, but seconds? At least, I think perf_counter is seconds.

However, I think ms is probably better, so maybe change time_for_span to report ms?